src/style.css

```diff
@@ -1 +1,35 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+}
+
+.stars--1:nth-child(-n + 1) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--2:nth-child(-n + 2) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--3:nth-child(-n + 3) {
+  background-image: url('images/star-active.svg');
+}
```

Comment:
Change the selector to `.stars--3 .stars__star:nth-child(-n + 3)` to apply the active image to the first three stars within the `.stars--3` container.